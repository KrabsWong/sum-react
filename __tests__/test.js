// import { renderHook, act } from '@testing-library/react-hooks'
// import BaseButton from '../components/BaseButton'
// import BaseModal from '../components/BaseModal'
const { renderHook, act } = require('@testing-library/react-hooks');
const BaseButton = require('../components/BaseButton');
const BaseModal = require('../components/BaseModal');

describe('BaseButton测试', () => {
  it('测试BaseButton', () => {
    const { result } = renderHook(() => BaseButton());
    act(() => {
      result.current.handleClick();
    });
    expect(result.current.handleClick()).toHaveBeenCalled();
  });

  it('测试BaseModal', () => {
    const { result } = renderHook(() => BaseModal());
    act(() => {
      result.current.handleClick();
    });
    expect(result.current.handleClick()).toHaveBeenCalled();
  });
});
