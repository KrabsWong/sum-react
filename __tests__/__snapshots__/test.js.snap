// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton BaseModal 1`] = `ShallowWrapper {}`;

exports[`BaseButton should have rendered BaseButton 1`] = `ShallowWrapper {}`;

exports[`BaseButton should have rendered BaseModal 1`] = `ShallowWrapper {}`;

exports[`BaseButton should have rendered button 1`] = `ShallowWrapper {}`;
